content/docs/user-guide/platforms/android/generating_android_project_windows.md

```diff
@@ -0,0 +1,139 @@
+
+**Source and Build directories**
+
+```
+SET O3DE_ENGINE_PATH=D:\github\o3de
+SET O3DE_PROJECT_PATH=D:\github\o3de-atom-sampleviewer
+SET O3DE_BUILD_ROOT=D:\build_asv
+```
+
+**Key Store Settings**
+
+```
+set O3DE_ANDROID_SIGNCONFIG_FILE=%O3DE_BUILD_ROOT%\o3de.keystore
+set O3DE_ANDROID_SIGNCONFIG_STORE_PASSWORD=o3de_android
+set O3DE_ANDROID_SIGNCONFIG_KEY_ALIAS=o3de_key
+set O3DE_ANDROID_SIGNCONFIG_KEY_PASSWORD=o3de_android
+set O3DE_ANDROID_SIGNCONFIG_KEY_SIZE=2048
+set O3DE_ANDROID_SIGNCONFIG_VALIDITY_DAYS=10000
+set O3DE_ANDROID_DN="cn=atom-sample-viewer, ou=o3de, o=LF, c=US"
+```
+
+{{< note >}}
+While have signing keys is required to deploy to an android device, this step is optional. If you already have an existing JKS file generated earlier, and know the values for the store password, key password, alias, etc, then they can be used in place of these values. 
+{{< /note >}}
+
+
+**Android Project Generation Settings**
+
+```
+set O3DE_ANDROID_SDK_PATH=C:\Android\android-sdk
+set O3DE_ANDROID_NDK_VERSION=22.*
+set O3DE_ANDROID_SDK_API_LEVEL=29
+set O3DE_ANDROID_ASSET_MODE=LOOSE
+set O3DE_ANDROID_DEPLOY_TYPE=BOTH
+```
+
+{{< note >}}
+The `O3DE_ANDROID_SDK_PATH` represents a specific location based on where you specify it in Android Studio or where you download and unpack the Android SDK Tools package
+{{< /note >}}
+
+<br/>
+
+## Steps
+
+{{< note >}}
+Steps 1-3 is only needed if the windows build of O3DE has not been completed yet. These steps describe the minimal build necessary in order to process assets for Android and the game project.
+{{< /note >}}
+
+
+**Step 1. Generate the cmake build directory for windows / vs2019 (for the android assets)**
+
+```
+cmake -B %O3DE_BUILD_ROOT%\windows -S %O3DE_ENGINE_PATH% -G"Visual Studio 16 2019" -DLY_PROJECTS=%O3DE_PROJECT_PATH% -DLY_UNITY_BUILD=ON
+```
+
+**Step 2. Build the Asset Processor Batch project in profile mode to process the assets**
+
+```
+cmake --build %O3DE_BUILD_ROOT%\windows --config profile --target AssetProcessorBatch -- /m /nologo
+```
+
+{{< note >}}
+This step will only build 'AssetProcessorBatch' and the necessary dependent modules. You can alternatively build the entire suite of tools, libraries, etc for windows by replacing `AssetProcessorBatch` with `ALL_BUILD`, or alternatively build from Visual Studio directly.
+{{< /note >}}
+
+**Step 3. Run Asset Processor Batch for android and Atom Sample Viewer**
+
+```
+%O3DE_BUILD_ROOT%\windows\bin\profile\AssetProcessorBatch.exe  --platforms=android --project-path %O3DE_PROJECT_PATH%
+```
+
+**Step 4. Generate a key store to be able to self-sign an APK**
+
+```
+keytool -genkey -keystore %O3DE_ANDROID_SIGNCONFIG_FILE% -storepass %O3DE_ANDROID_SIGNCONFIG_STORE_PASSWORD% -alias %O3DE_ANDROID_SIGNCONFIG_KEY_ALIAS% -keypass %O3DE_ANDROID_SIGNCONFIG_KEY_PASSWORD% -keyalg RSA -keysize %O3DE_ANDROID_SIGNCONFIG_KEY_SIZE% -validity %O3DE_ANDROID_SIGNCONFIG_VALIDITY_DAYS% -dname %O3DE_ANDROID_DN%
+```
```

Comment:
- Marked this as optional and provided a clear alternative step. 
```suggestion
2. (Optional) Generate a KeyStore to allow APK signing. If you don't set up a signing key at this time, you can also set it up through Android Studio after generating the Android project.

    ```
    keytool -genkey -keystore %O3DE_ANDROID_SIGNCONFIG_FILE% -storepass %O3DE_ANDROID_SIGNCONFIG_STORE_PASSWORD% -alias %O3DE_ANDROID_SIGNCONFIG_KEY_ALIAS% -keypass %O3DE_ANDROID_SIGNCONFIG_KEY_PASSWORD% -keyalg RSA -keysize %O3DE_ANDROID_SIGNCONFIG_KEY_SIZE% -validity %O3DE_ANDROID_SIGNCONFIG_VALIDITY_DAYS% -dname %O3DE_ANDROID_DN%
    ```
```
```